src/components/Projects/ProjectBlock.tsx

```diff
@@ -0,0 +1,124 @@
+import { BodyText, HeaderText, PrimaryAccentText } from 'components/Text'
+import { JSX } from 'preact/jsx-runtime'
+import { useRef } from 'preact/hooks'
+import Button from 'components/Button'
+import Tilt from 'react-parallax-tilt'
+import classnames, {
+  TTextColor,
+  alignItems,
+  display,
+  dropShadow,
+  flexDirection,
+  gap,
+  inset,
+  justifyContent,
+  margin,
+  position,
+  rotate,
+  scale,
+  space,
+  width,
+  zIndex,
+} from 'classnames/tailwind'
+
+type Element = JSX.Element
+
+const wrapper = (reverse?: boolean, centered?: boolean) =>
+  classnames(
+    position('relative'),
+    display('flex'),
+    gap('gap-8'),
+    flexDirection(
+      { 'lg:flex-row-reverse': reverse, 'flex-col-reverse': reverse },
+      'flex-col',
+      'lg:flex-row'
+    ),
```

Comment:
It will reverse the row, we need it here:
![image](https://user-images.githubusercontent.com/49658988/192354662-9b68caf5-5a4b-419e-b9fe-a1ee36dd97dc.png)